src/components/tooltip/tooltip.js

```diff
@@ -75,6 +75,7 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $interpolate,
 
   function linkFunc(scope, element, attr) {
     // Set constants.
+    var tooltipId = 'tooltip-' + $mdUtil.nextUid();
```

Comment:
How about prefixing the id with `md-`? (As same as in the `datepicker` [here](https://github.com/angular/material/blob/342ee53363e06a46958541eac980e5ff3d62029d/src/components/datepicker/js/datepickerDirective.js#L331))